runners/flink/job-server-container/flink_job_server_container.gradle

```diff
@@ -23,6 +23,7 @@
  */
 
 apply plugin: 'org.apache.beam.module'
+apply plugin: 'base'
```

Comment:
When is it necessary to apply the base plugin? What error do we get without it?

Reply:
see failure on PR https://github.com/apache/beam/pull/7787 python_PreCommit 

https://scans.gradle.com/s/7vd7a5pylnrmu

Running  `./gradlew -p runners/flink/job-server-container/  copyDockerfileDependencies`
with gradle 5.0 gives a different error message

> FAILURE: Build failed with an exception.
What went wrong:
Could not determine the dependencies of task ':beam-runners-flink_2.11-job-server-container:copyDockerfileDependencies'.
 Could not resolve all dependencies for configuration ':beam-runners-flink_2.11-job-server-container:dockerDependency'.
    The destinationDir property must be set. Please apply the base plugin or set it explicitly.

The destinantionDIr property is missing on some abstractarchivetask, which is handled by the base plugin. I honestly do not yet fully understand the changes caused by that provider api. To much internal magic going on which I do not fully understand, unfortunately. I just trust gradle in what it is telling me...

I m unsure, whether this is a problem caused by beams crossreferencing setup, which is more complex than 'usual' gradle projects. Or it might be some issue with gradle itself. AFAIU, the provider api is rather new and might have some edges causing trouble here.